extensions/amp-pan-zoom/validator-amp-pan-zoom.protoascii

```diff
@@ -0,0 +1,57 @@
+#
+# Copyright 2018 The AMP HTML Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the license.
+#
+tags: {  # amp-pan-zoom
+  html_format: AMP
+  html_format: EXPERIMENTAL
```

Comment:
I don't think you want `EXPERIMENTAL` since it's not on the tag itself. If you think otherwise, lets talk offline.

Reply:
Done